pkg/keyspace/tso_keyspace_group.go

```diff
@@ -112,6 +113,14 @@ func (m *GroupManager) Bootstrap() error {
 
 	m.Lock()
 	defer m.Unlock()
+
+	// If the etcd client is not nil, start the watch loop.
+	if m.client != nil {
+		m.nodesBalancer = balancer.GenByPolicy[string](m.policy)
```

Comment:
```suggestion
		m.nodesBalancer = balancer.GenByPolicy[string](m.policy)
		m.serviceRegistryMap = make(map[string]string)
```
because of https://github.com/tikv/pd/pull/6346 being merged